.gitignore

```diff
@@ -65,6 +65,7 @@ Thumbs.db
 */overlays
 *~
 .directory
+assembly/assembly-dashboard-war/overlays/
```

Comment:
Maybe just modify L65 to `**/overlays` ?

Reply:
I was thinking about that, but it felt too wide for me. Do you think it won't cause issues when we would need to create `overlays` folder anywhere? (probably not and my worries are irrelevant here)

Reply:
dont think it's a big issue... btw we can do `assembly/**/overlays` to narrower scope

Reply:
updated